internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -12,12 +12,23 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+type MaturityStatus string
+
+const (
+	MaturityStatusTechPreview MaturityStatus = "preview"
+	MaturityStatusStable      MaturityStatus = "stable"
+)
+
 type KafkaInstanceType struct {
 	Id          string              `yaml:"id"`
 	DisplayName string              `yaml:"display_name"`
 	Sizes       []KafkaInstanceSize `yaml:"sizes"`
 }
 
+func getValidMaturityStates() []MaturityStatus {
```

Comment:
In https://issues.redhat.com/browse/MGDSTRM-8476 more states are listed. Is having this two values only for now intentional?

Reply:
As per Jira: `For now, preview and stable will suffice.`. No harm in adding the others already.

Reply:
:+1: we can leave it as it is for now then